README.md

```diff
@@ -1 +1,67 @@
-# ezra_portfolio
\ No newline at end of file
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [HTML](#tech-stack)
+    - [CSS](#key-features)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
```

Comment:
- [ ] Your README file is missing a lot of sections. Please make sure to include all the sections in the [Microverse Readme template](https://github.com/microverseinc/readme-template) :100:  Every section of the readme template is required with the exception of the FAQ section which is optional. So please make sure to add all the missing sections to your project and fill in all the necessary contents :+1: